frontend/src/components/not-found/not-found.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, Container, Row, Col } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { getAllowedClasses } from 'helpers/dom/dom';
+import { AppRoute } from 'common/enums/enums';
+import styles from './styles.module.scss';
+
+const NotFound: React.FC = () => {
+  return (
+    <Container
+      className={`${getAllowedClasses(
+        styles.notFoundWrapper,
+      )} d-flex flex-column`}
+    >
+      <Row className="vh-100">
+        <Col sm={10} md={8} lg={6} className="mx-auto d-table vh-100">
+          <div className="d-table-cell align-middle">
```

Comment:
Wouldn't flex-box be better to align something to the center
```suggestion
<Col sm={10} md={8} lg={6} className="d-flex flex-column justify-content-center vh-100">
```